source/addonHandler/__init__.py

```diff
@@ -1014,6 +1014,13 @@ class AddonManifest(ConfigObj):
 		input = boolean(default=true)
 		output = boolean(default=true)
 
+# Symbol Pronunciation
+[symbolDictionaries]
+	# The key is the symbol dictionary file name (not the full path)
+	[[__many__]]
+		displayName = string()
```

Comment:
can displayName be made translatable?

Reply:
See above, it is translatable in the locale specific manifests